src/loaders/JsonImageInfoLoader.ts

```diff
@@ -130,6 +135,10 @@ class JsonImageInfoLoader implements IVolumeLoader {
     this.jsonInfo = await this.getJsonImageInfo(loadSpec);
     const imageInfo = convertImageInfo(this.jsonInfo);
 
+    this.cacheStore = this.cache?.addVolume(imageInfo.numChannels, Math.max(imageInfo.times, this.urls.length), [
+      new Vector3(imageInfo.subregionSize.x, imageInfo.subregionSize.y, imageInfo.subregionSize.z),
+    ]);
+
```

Comment:
If `createVolume` is called multiple times, will this add duplicates of the same volume to `cache`?

Reply:
Hm, that might be out of scope of this PR but it's something for us to worry about for sure.